ansible/files/admin_api_scripts/pg_upgrade_prepare.sh

```diff
@@ -0,0 +1,7 @@
+#! /usr/bin/env bash
+
+set -euo pipefail
+
+systemctl stop postgresql
+umount /data
```

Comment:
I guess this upgrade isn't going to work for really old projects, since they didn't use this mount point?

Reply:
This runs on the "new" instance - the upgraded one which would replace the database instance. 
Could use more comments throughout these scripts to describe what each one does